content/en/docs/quickstart.md

```diff
@@ -43,86 +43,96 @@ notation list --plain-http $IMAGE
 
 ```console
 notation list --plain-http $IMAGE
 ```
 
-Confirm there is one listed, for example:
+Confirm there is one signature, for example:
 
 ```output
 $ notation list --plain-http $IMAGE
-sha256:1234567890abcdef1234567890abcdef1234567890abcdef1234567890abcdef
+localhost:5000/net-monitor@sha256:073b75987e95b89f187a89809f08a32033972bb63cda279db8a9ca16b7ff555a
+└── application/vnd.cncf.notary.v2.signature
+    └── sha256:ba3a68a28648ba18c51a479145fca60d96b43dc96c6ab22f412c89ac56a9038b
 ```
 
-## Verify the signature of an image
-
-Use `notification verify` to verify any signatures on your container image.
-
-```console
-notation verify --plain-http $IMAGE
+## Create a trust policy
+
+In order to verify the container image, you need to configure the trust policy to specify trusted identities which sign the artifacts, and level of signature verification to use. See [trust policy spec](https://github.com/notaryproject/notaryproject/blob/main/trust-store-trust-policy-specification.md#trust-policy) to understand more about trust policy.
+
+Create a json file named `trustpolicy.json` with the following content:
+
+```json
+{
+    "version": "1.0",
+    "trustPolicies": [
+        {
+            "name": "wabbit-networks-images",
+            "registryScopes": [ "*" ],
+            "signatureVerification": {
+                "level" : "strict" 
+            },
+            "trustStores": [ "ca:wabbit-networks.io" ],
+            "trustedIdentities": [
+                "*"
+            ]
+        }
+    ]
+}
 ```
 
-Confirm your container image fails validation. For example:
-
-```output
-$ notation verify --plain-http $IMAGE
-Error: trust certificate not specified
-2022/09/30 15:24:14 trust certificate not specified
-```
+For a Linux user, store file `trustpolicy.json` under directory `$HOME/.config/notation/`.
 
-This verification fails because you have not trusted the public key for your container image.
+For a Mac user, store file `trustpolicy.json` under directory `$HOME/Library/Application Support/notation/`.
 
-## Add a trusted public key
+For a Window user, store file `trustpolicy.json` under directory `C:\Users\<username>\AppData\Roaming\notation\`.
 
-Add the public from the certificate you generated earlier as a trusted certificate.
+Once the trust policy file is executed, it creates one trust policy named `wabbit-networks-images`. The trust policy with `registryScopes` set to `*` applies to all the artifacts in any registries. The level of signatureVerification is `strict`, which enforces a full validation. Any validation failure will fail the whole signature verification process. The trust store used for this policy is named `wabbit-networks.io` of type `ca`, which is created in previous step. See [trust policy spec](https://github.com/notaryproject/notaryproject/blob/main/trust-store-trust-policy-specification.md#trust-policy-properties) to fine tune the policies for specific security requirements.
```

Comment:
Same as above. We should use a relative link here. You can replace it with [trust policy spec]({{< ref "/docs/concepts/trust-store-trust-policy-specification/#trust-policy-properties" >}}).